### ISSUE TITLE
og-image service does not show Chinese characters properly

### ISSUE THREAD

**Issue by wizyoung** [yokinist/yokinist.me#1]
Hi, thanks for your great work. I find a bug that your forked og-image service may swallow some Chinese characters.
![2021-10-03 19 01 41](https://user-images.githubusercontent.com/13296106/135750829-ef4f539e-108e-4eb5-83eb-ec5f228bc0ed.jpg)
 As you see, some characters do not show.

**Comment by yokinist** [comment]
@wizyoung Thanks for the report! I'll check it 👀

**Comment by yokinist** [comment]
https://github.com/yokinist/og-image

**Comment by yokinist** [comment]
@wizyoung 

I fixed the problem you reported.

If you add `&lang=zh` to the end of the URL of the OGP image, it will be rendered normally in Chinese 🇨🇳

![image](https://user-images.githubusercontent.com/19779874/136026811-cb3ca817-e1f9-4004-b4a9-86fb2cfbb690.png)

https://simple-og-image.vercel.app/%E4%B8%80%E4%B8%AA%E8%A7%A3%E5%86%B3%E9%80%9A%E7%94%A8%E5%89%AA%E8%B4%B4%E6%9D%BF%E5%A4%B1%E6%95%88%E7%9A%84%E6%9B%B2%E7%BA%BF%E6%95%91%E5%9B%BD%E6%96%B9%E6%A1%88.png?siteTitle=example.com&background=%23090719&foreground=%23FFFFFF&md=1&fontSize=100px&isTwitter=1&lang=zh

**Comment by wizyoung** [comment]
Thanks for your fix, but seems it still has some problems.
The link is https://nobelium-by0k6wnjf-wizyoung.vercel.app/yet-another-method-to-fix-universal-clipboard?lang=zh, shows as 
<img width="463" alt="image" src="https://user-images.githubusercontent.com/13296106/136056411-5bd120e3-846c-441a-84d2-2ee91934264a.png">
The above og image is generated by your https://simple-og-image.vercel.app service. 
But if I switch back to https://og-image-craigary.vercel.app, then the og image goes normal:
<img width="463" alt="image" src="https://user-images.githubusercontent.com/13296106/136057067-1f0155d8-1023-4a22-8342-88e335c22dc4.png">

I don't the reason behind as I know little about js/ts. My fork version of your nobelium is https://github.com/wizyoung/nobelium

**Comment by yokinist** [comment]
@wizyoung Does it work this solution?
https://github.com/wizyoung/nobelium/pull/1

**Comment by wizyoung** [comment]
It seems Notion changed their api hours ago? I got "NotionAPI collectionQuery error Response code 400 (Bad Request)" error now

**Comment by wizyoung** [comment]
I'm waitting for the update now :) https://github.com/NotionX/react-notion-x/issues/139

**Comment by yokinist** [comment]
I'm facing the same error... 😭
I'll check again later. Perhaps a temporary solution would be to downgrade the package version.

https://github.com/yokinist/yokinist.me/pull/2

**Comment by wizyoung** [comment]
I think switching to notion official api would be a better choice when it releases the first stable version.

**Comment by wizyoung** [comment]
Now notion-client 4.9.2 solved the NotionAPI breakdown.
But your og-image patch stills not works 😂 I tried several apps to find that only twitter can parse the chinese characters correctly. That's quite wierd. I also checked the og-image here: https://opengraphcheck.com/result.php?url=https%3A%2F%2Fnobelium-htglb69h8-wizyoung.vercel.app%2Fyet-another-method-to-fix-universal-clipboard%3Flang%3Dzh. Maybe there is something wrong with the locale?
